arch/xtensa/soc/esp32/esp32-mp.c

```diff
@@ -162,6 +196,8 @@ static void appcpu_start(void)
 	 * the CPU, but this is how they do it...
```

Comment:
This is because the APP CPU does not actually start at the address set by `esp32_rom_ets_set_appcpu_boot_addr`. APP CPU runs the same ROM code as the PRO CPU (set up basic CPU state, initialize the stack and per-cpu .bss/.data used by ROM code), but then goes into a loop waiting for the entry address to become non-zero. I think the other order of doing things would also work.